### PR TITLE
Home dashboard

### PR DESCRIPTION
**Pull request by angel-afonso**
# Added some statistics to home screen

Reason:
I was feeling anxiety when reading the "test" text on the home screen

Before pull request:
A "test" text is showed in the first screen (terrible)

After pull request:
A cards with some statistics is displayed

### CODE REVIEW

File: src/redux/reducers/sessionReducer.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ export default function sessionReducer(state = initialState, action) {
         loading: true,
       };
     case sessionActions.LOGIN:
+      console.trace();
```
Comment (HectorZR) [inline src/redux/reducers/sessionReducer.js:13]: You forgot to delete this `console.trace()`

Comment (HectorZR) [inline src/redux/reducers/sessionReducer.js:13]: You forgot to delete this `console.trace()`

**Review by HectorZR — CHANGES_REQUESTED** [review]
Just a minor fix, anything else looks really good!

File: src/redux/reducers/sessionReducer.js (hunk 1 above)
Comment (angel-afonso) [reply src/redux/reducers/sessionReducer.js]: Done

**Review by HectorZR — APPROVED** [review]
Everything looks good to me!

**Pull request merged**